dashboard/app/controllers/projects_lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsListsController < ApplicationController
+  include LevelsHelper
+
+  def index
+    section_id = params[:section_id]
+    section_owner_id = Section.find(section_id).user_id
+    raise "section can only be accessed by its owner" unless current_user && current_user.id == section_owner_id
+
+    @projects_list_data = []
+    Section.find(section_id).students.each do |student|
```

Comment:
nit: this `Section.find` would have been a 2nd DB call (duplicate with line 6). Better to grab the section once and reuse it.